poly-commit/src/utils.rs

```diff
@@ -70,21 +78,37 @@ impl<F: Field> Matrix<F> {
         );
 
         // TODO more efficient to run linearly?
-        let entries: Vec<Vec<F>> = (0..n)
+        let rows: Vec<Vec<F>> = (0..n)
             .map(|row| (0..m).map(|col| entry_list[m * row + col]).collect())
             .collect();
 
-        Self { n, m, entries }
+        Self { n, m, rows }
+    }
+
+    /// Returns self as a list of rows
+    pub(crate) fn rows(&self) -> &Vec<Vec<F>> {
+        &self.rows
+    }
+
+    /// Returns the entry in position (i, j). **Indexing starts at 0 in both coordinates**,
+    /// i.e. the first element is in position (0, 0) and the last one in (n - 1, j - 1),
+    /// where n and m are the number of rows and columns, respectively.
+    ///
+    /// Index bound checks are waived for efficiency and behaviour under invalid indexing is undefined
+    #[cfg(test)]
+    pub(crate) fn entry(&self, i: usize, j: usize) -> F {
+        self.rows[i][j]
```

Comment:
this doesn't seem to be used

Reply:
Yes, it is not used in Hyrax and in Ligero, it is only for testing. We can remove it.